docs/ota-client-guide/modules/ROOT/pages/device-monitoring-with-zabbix.adoc

```diff
@@ -0,0 +1,227 @@
+
+=== Table of contents
+* <<Install and setup Zabbix server on Ubuntu 16/18>>
+* <<Build an OTA image for QEMU>>
+* <<Run a QEMU image>>
+* <<Run a QEMU image in the background mode>> 
+* <<Add a template and configure a host for Zabbix server>>
+
+
+=== Install and setup Zabbix server on Ubuntu 16/18
```

Comment:
```suggestion
== Install and setup Zabbix server on Ubuntu 16/18
```

Heading levels should always be sequential--never skipping from `==` to `====`, for example.